bundles/org.openhab.binding.intesis/README.md

```diff
@@ -1,62 +1,65 @@
+| Thing       | Thing Type | Description                                 |
+|-------------|------------|---------------------------------------------|
+| intesisHome | Thing      | Represents a single IntesisHome WiFi device |
+| intesisBox  | Thing      | Represents a single IntesisBox WiFi device  |
 
 ## Discovery
 
 Intesis devices do not support auto discovery.
 
 ## Thing Configuration
 
-The binding needs two configuration parameters.
+The binding uses the following configuration parameters.
 
-| Parameter | Description                                       |
-|-----------|---------------------------------------------------|
-| ipAddress | IP-Address of the              device             |
-| password  | Password to login to the local webserver of device |
+| Parameter | Valid for ThingType | Description                                                    |
+|-----------|---------------------|----------------------------------------------------------------|
+| ipAddress | Both                | IP-Address of the device                                       |
+| password  | IntesisHome         | Password to login to the local webserver of IntesisHome device |
+| port      | IntesisBox          | TCP port to connect to IntesisBox device, defaults to 3310     |
 
 
 ## Channels
 
-| Channel ID         | Item Type          | Description                                 | Possible Values           |
-|--------------------|--------------------|---------------------------------------------|---------------------------|
-| power              | Switch             | Turns power on/off for your climate system. | ON, OFF                   |
-| mode               | String             | The heating/cooling mode.                   | AUTO,HEAT,DRY,FAN,COOL    |
-| fanSpeed           | String             | Fan speed (if applicable)                   | AUTO,1-10                 |
-| vanesUpDown        | String             | Control of up/down vanes (if applicable)    | AUTO,1-9,SWING,SWIRL,WIDE |
-| vanesUpDown        | String             | Control of left/right vanes (if applicable) | AUTO,1-9,SWING,SWIRL,WIDE |
-| targetTemperature  | Number:Temperature | The currently set target temperature.       |                           |
-| ambientTemperature | Number:Temperature | (Readonly) The ambient air temperature.     |                           |
-| outdoorTemperature | Number:Temperature | (Readonly) The outdoor air temperature.     |                           |
+| Channel ID         | Item Type          | Description                                 | Possible Values             |
+|--------------------|--------------------|---------------------------------------------|-----------------------------|
+| power              | Switch             | Turns power on/off for your climate system. | ON, OFF                     |
+| mode               | String             | The heating/cooling mode.                   | AUTO,HEAT,DRY,FAN,COOL      |
```

Comment:
Is the "if applicable" missing for the mode Channel and some others?

Reply:
Done